### ISSUE TITLE
[Bug Report] video output not being displayed on Android emulator

### ISSUE THREAD

**Issue by Holofox** [alexmercerind/media_kit#149]
Hello!

I ran the example on an android emulator but I see a black background instead of a video. I tried different versions of emulators, but it did not help. 
```
Pixel_4_API_31      • Pixel 4 API 31   • Google  • android
Pixel_5_API_30      • Pixel 5 API 30   • Google  • android
Resizable_API_33    • Resizable API 33 • Generic • android
```

```
[✓] Flutter (Channel stable, 3.7.11, on macOS 13.0 22A380 darwin-x64)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
[✓] Chrome - develop for the web
[✓] IntelliJ IDEA Ultimate Edition (version 2022.2.5)
[✓] Connected device (3 available)
[✓] HTTP Host Availability
```

| 01  |   06 |
| ------------- | ------------- |
| ![Screenshot_1682062339](https://user-images.githubusercontent.com/19191492/233572766-e0fb2d76-a4ee-4cfc-aa75-a1851495e09d.png) |  ![Screenshot_1682062328](https://user-images.githubusercontent.com/19191492/233572779-d2713d12-9f32-4464-aa79-7fc9c3ac6dce.png)  |

media_kit 0.0.5+2

**Comment by alexmercerind** [comment]
Can you try the demo application from README from real device for confirmation?

**Comment by Holofox** [comment]
> Can you try the demo application from README on a real device for confirmation?

@alexmercerind, I checked, on a real device it works fine.

**Comment by alexmercerind** [comment]
Hi @Holofox, video rendering in emulator should be fixed in #148.

Thanks (a confirmation will be nice)!

**Comment by alexmercerind** [comment]
Closing as the latest release resolves this.
